### ISSUE TITLE
frontend-backend connection: get existing schemas

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#63]
## User story
1. As a {HR-employee}
2. I want / need {the DIdentity backend and frontend to work together}
3. So that {the existing schemas can actually be displayed to me in a schema-overview-page}

## Acceptance criteria
* Overview over process:
  *  First, existing schemas are provided to the frontend #54 
  * Second, retrieved schemas are displayed in schema-overview page #52 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
